### PR TITLE
fix(auth): FlutterFactory for Flutter constructors

### PR DESCRIPTION
**Pull request by tylerjroach**
- [ ] PR title and description conform to [Pull Request](https://github.com/aws-amplify/amplify-android/blob/main/CONTRIBUTING.md#pull-request-guidelines) guidelines.

*Issue #, if available:*

*Description of changes:*

*How did you test these changes?*
(Please add a line here how the changes were tested)

- [ ] Added Unit Tests
- [ ] Added Integration Tests

*Documentation update required?*
- [ ] No
- [ ] Yes (Please include a PR link for the documentation update)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/helpers/FlutterFactory.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package com.amplifyframework.auth.cognito.helpers
+
+import com.amplifyframework.auth.AWSCredentials
+import com.amplifyframework.auth.cognito.AWSCognitoAuthSession
+import com.amplifyframework.auth.cognito.AWSCognitoUserPoolTokens
+import com.amplifyframework.auth.result.AuthSessionResult
+
+/**
+ * Helper class that allows Amplify Flutter dev team to create necessary return types with internal constructors.
+ * This class should not be consumed outside of the Amplify Flutter development team and should not be considered
```
Comment (sktimalsina) [inline aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/helpers/FlutterFactory.kt:10]:
```suggestion
 * Helper class that allows Amplify Flutter to create necessary return types with internal constructors.
 * This class should not be consumed outside of the Amplify Flutter and should not be considered
```

**Review by dnys1 — APPROVED** [review]
Thanks! LGTM

**Review by sktimalsina — COMMENTED** [review]
Suggested changes

**Review by div5yesh — APPROVED** [review]
LGTM

File: aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/helpers/FlutterFactory.kt (hunk 1 above)
Comment (tylerjroach) [reply aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/helpers/FlutterFactory.kt]: Resolving with additional removal of extra "the"

File: aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/helpers/FlutterFactory.kt [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package com.amplifyframework.auth.cognito.helpers
+
+import com.amplifyframework.auth.AWSCredentials
+import com.amplifyframework.auth.cognito.AWSCognitoAuthSession
+import com.amplifyframework.auth.cognito.AWSCognitoUserPoolTokens
+import com.amplifyframework.auth.result.AuthSessionResult
+
+/**
+ * Helper class that allows Amplify Flutter to create necessary return types with internal constructors.
```
Comment (palpatim) [inline aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/helpers/FlutterFactory.kt:9]: See https://github.com/aws-amplify/amplify-swift/blob/7f094a9e88612b5b8fda1bce01ce71e7f1915053/Amplify/Categories/DataStore/Model/Internal/Schema/ModelSchema%2BDefinition.swift#L11-L12 for example message from Swift

Comment (palpatim) [reply aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/helpers/FlutterFactory.kt:9]: Also, "should not be considered" is probably fragmentary. :)

**Pull request merged**